### ISSUE TITLE
Doesn't detect a symbol from libX11

### ISSUE THREAD

**Issue by lvc** [universal-ctags/ctags#1198]
The latest universal ctags 015eae73de29895b494528615ac639b856fe2e3a doesn't detect function `XESetCopyEventCookie` from libX11-1.3 library (include/X11/Xlibint.h):
    
    extern Bool (*XESetCopyEventCookie(
        Display*            /* display */,
        int                 /* extension */,
        Bool (*) (
                   Display*                 /* display */,
                   XGenericEventCookie*     /* in */,
                   XGenericEventCookie*     /* out */
                 )          /* proc */
    ))(
        Display*, XGenericEventCookie*, XGenericEventCookie*
    );

The command:

    ctags -x --c-kinds=fpvx include/X11/Xlibint.h|grep XESetCopyEventCookie

returns empty output.

Thank you.

**Comment by masatake** [comment]
```
int (* f (char c)) (void * a);

static int g (void *a)
{
	return 1;
}

int (* f (char c)) (void * a)
{
	return g;
}
```

**Comment by masatake** [comment]
BTW, OldC captures them well. 
```
 ./ctags -o - --languages=+OldC --language-force=OldC --OldC-kinds=fpvx x.c 
f	x.c	/^int (* f (char c)) (void * a)$/;"	f
f	x.c	/^int (* f (char c)) (void * a);$/;"	p	file:
g	x.c	/^static int g (void *a)$/;"	f	file:
```

**Comment by pragmaware** [comment]
Fixed via #1207

**Comment by lvc** [comment]
Verified. Thank you!
